ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -1,11 +1,26 @@
+import {mount} from 'enzyme';
 import * as React from 'react';
 
-import {mount} from 'enzyme';
+import {cohortReviewStore} from 'app/services/review-state.service';
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {currentWorkspaceStore} from 'app/utils/navigation';
+import {CohortAnnotationDefinitionApi, CohortReviewApi} from 'generated/fetch';
+import {CohortAnnotationDefinitionServiceStub} from 'testing/stubs/cohort-annotation-definition-service-stub';
+import {CohortReviewServiceStub, cohortReviewStubs} from 'testing/stubs/cohort-review-service-stub';
+
+import {workspaceDataStub} from 'testing/stubs/workspaces-api-stub';
 import {HelpSidebar} from './help-sidebar';
 
-describe('SidebarContent', () => {
+describe('HelpSidebar', () => {
+  beforeEach(() => {
+    registerApiClient(CohortReviewApi, new CohortReviewServiceStub());
+    registerApiClient(CohortAnnotationDefinitionApi, new CohortAnnotationDefinitionServiceStub());
+    currentWorkspaceStore.next(workspaceDataStub);
+    cohortReviewStore.next(cohortReviewStubs[0]);
+  });
+
   it('should render', () => {
-    const wrapper = mount(<HelpSidebar location='data' />);
+    const wrapper = mount(<HelpSidebar helpContent='data' hideSidebar={0} />);
```

Comment:
Can we add another test that checks whether the content changes from page to page?

Also, we should test that the sidebar is hidden if hide is set to true.